vitrina/uapi/templates/agents/agent_detail.html

```diff
@@ -235,11 +235,76 @@
     {% endif %}
     <hr>
 
-    <p class="is-flex is-justify-content-flex-end">
-        <a href="{% url 'agent-list' organization.pk %}">
-            ← {% translate "Grįžti į agentus" %}
-        </a>
-    </p>
+{% if request_history %}
+    <p class="is-size-4 has-text-weight-bold mb-3">{% translate "Užklausų istorija" %}</p>
+    <div class="mb-5">
+        <table class="table is-fullwidth is-hoverable">
+            <thead style="background-color: #f5f5f5;">
+            <tr>
+                <th style="border-bottom: none; border-right: 1px solid #ccc;">
```

Comment:
Wasn't the table inside agent_list.html not enough? I am asking because I really dislike the inline `style` attributes for tags:D and also to keep up the style through-out different pages.